main.py

```diff
@@ -9,7 +9,7 @@
 import marketplace
 import pickle
 from log import *
-from settings import *
+from settings_default import *
```

Comment:
In fact, settings_default should not be imported. Instead, one must copy `settings_default.py` as `settings.py` and change the default values before launching the app.

If this was a real project, this would be explained in the documentation, the readme or in an installation tutorial. But 
🤷